src/main/styles/scss/_header.scss

```diff
@@ -1,63 +1,64 @@
 nav {
-    position: fixed;
-    z-index: 1;
-    width: 100%;
-    height: 52px;
-    background: rgb(20,20,20);
-    color: aliceblue;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    transition: 0.4s;
-  }
-  nav > a {
-    display: block;
-    height: 75%;
-    margin: auto 3rem;
-  }
-  a > img {
-    display: block;
-  }
-  ul {
-    list-style-type: none;
-    display: flex;
-    font-family: Arial, Helvetica, sans-serif;
-    text-decoration: none;
-  }
-  li {
-    margin-right: 3rem;
-  }
-  li > a {
-    color: aliceblue;
-    text-decoration: none;
-    transition: 0.4s;
-  }
-  li > a:hover {
-    text-decoration: none;
-    color: rgb(99, 98, 94);
-  }
-  .burger {
-    display: none;
-    position: relative;    
-    z-index: 2;
-    height: 60%;
-    -webkit-user-select: none;
-    user-select: none;
-    flex-direction: column;
-    justify-content: center;
-    margin-right: 3rem;
-    cursor: pointer;
-  }
-  .burger span {
-    display: block;
-    width: 30px;
-    height: 2px;
-    margin-top: 3px;
-    margin-bottom: 3px;
-    position: relative;
-    top: auto;
-    bottom: auto;
-    background: aliceblue ;
-    border-radius: 3px;
-    z-index: 2;
-  }
\ No newline at end of file
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  height: 52px;
+  background: rgb(0,0,0,99);
```

Comment:
Shouldn't it be rgba instead of rgb? And shouldn't this last parameter be between 0 and 1?

Reply:
Done